encrypted-media-respec.html

```diff
@@ -1542,17 +1542,27 @@
                     <li><p>Let <var>session id</var> be a unique <a def-id="session-id"></a> string.</p>
                       <p>If the result of running the <a def-id="is-persistent-session-type-algorithm"></a> algorithm on <var>session type</var> is <code>true</code>, the ID MUST be unique within the <a def-id="origin"></a> of this object's <a def-id="document-concept"></a> over time, including across Documents and browsing sessions.</p>
                     </li>
-                    <li><p>Let <var>message</var> be a license request for the <var>requested license type</var> generated based on the <var>sanitized init data</var>, which is interpreted per <var>initDataType</var>.</p>
+                    <li><p>Let <var>message</var> be a license request for the <var>requested license type</var> generated based on the <var>sanitized init data</var> interpreted per <var>initDataType</var>, or an individualization request.</p>
```

Comment:
This does not cover all use cases. Specifically, it doesn't cover the server certificate request. I think we need something more general that results in the requested license request or not.

I think we can cover all three known cases (license request, server certificate, individualization) by checking whether a license request can be generated at this time. Even the fourth theoretical case of a multi-round license request process could probably fall into a) generating some type of request or b) not being able to at this time.

For example:
* If a license request can be generated at this time, let _message_...{existing text}
* Otherwise:
   - Run the following steps:
      1. Let _message_ be a request that needs to be processed before generating a request.
      1. Let _message type_ be `<a def-id="message-type-individualization-request">` for xyz and `<a def-id="message-type-license-request"></a>` otherwise.
   - When the response to _message_ is provided in a subsequent call to `<a def-id="update"></a>`, the CDM MUST... {text below}<br/>{Issue box from below}